pkg/db/drivers/etcd/etcd.go

```diff
@@ -579,6 +579,35 @@ func (c *Client) GetPool(ctx *c.Context, polID string) (*model.StoragePoolSpec,
 	return pol, nil
 }
 
+//GetPoolsAz
+func (c *Client) GetAvailabilityZones(ctx *c.Context) ([]string, error) {
+	dbReq := &Request{
+		Url: urls.GenerateAZonesURL(urls.Etcd,""),
+	}
+	fmt.Println("GetAvailabilityZones dbReq:", dbReq)
```

Comment:
Do NOT print some internal message in the code, if the message is printed for debugging, please use `glog.Info()` instead.